[
  {
    "project": "python-telegram-bot/python-telegram-bot",
    "commit": "7e554584b117aae787bdb0b11d29d2358544a67b",
    "filename": "telegram/ext/conversationhandler.py",
    "filename_after_commit": "/home/chowyi/TypeAnnotation_Study/GitHub/python-telegram-bot-python-telegram-bot/telegram/ext/conversationhandler.py",
    "file_hunks_size": 12,
    "min_patch_found": false,
    "single_hunk": false,
    "fit_TFix": false,
    "delete_only_patch": false,
    "has_suppression_all_hunks": false,
    "full_warning_msg": "telegram/ext/conversationhandler.py:218:8 Incompatible variable type [9]: conversation_timeout is declared to have type `int` but is used as type `None`.",
    "message": " conversation_timeout is declared to have type `int` but is used as type `None`.",
    "rule_id": "Incompatible variable type [9]",
    "warning_line_no": 218,
    "warning_line": "        conversation_timeout: int = None,"
  }
]